verification/VerifyTasks.wdl

```diff
@@ -193,4 +193,72 @@ task CompareCompressedTextFiles {
     preemptible: 3
   }
 
+}
+
+task CompareLooms {
+
+  input {
+    File truth_loom
+    File test_loom
+    Float delta_threshold = 0.05
+
+    Int cpu = 3
+    String docker = "us.gcr.io/broad-dsp-gcr-public/base/python:3.9-debian"
+    Int disk_size_gb = ceil((size(truth_loom, "GiB") + size(test_loom, "GiB")) * 2) + 20
+    Int memory_mb = ceil(size(truth_loom, "MiB") + size(test_loom, "MiB") * 4) + 20000
+  }
+
+  command <<<
+  set -e
+  pip3 install scanpy loompy numpy pandas > /dev/null
+
+  python3 <<CODE
+  import sys
+  import scanpy
+  import numpy as np
+  
+
+  truth_loom = "~{truth_loom}"
+  test_loom="~{test_loom}"
+  threshold = "~{delta_threshold}"
+
+  test_loom = scanpy.read_loom(
+    truth_loom, obs_names="cell_names", var_names="gene_names"
+  )
+  truth_loom = scanpy.read_loom(
+      test_loom, obs_names="cell_names", var_names="gene_names"
+  )
+
+  print("foo")
+  truth_cells = np.array(test_loom.X.sum(axis=1)).flatten()
+  test_cells = np.array(truth_loom.X.sum(axis=1)).flatten()
+
+  differences = [
+    1 for (truth, test) in zip(truth_cells, test_cells) if (truth - test) != 0
+  ]
+
+  delta = len(differences) / len(truth_cells)
+
+  if delta < threshold:
```

Comment:
Why would we allow any differences?

Reply:
@khajoue2 suggested this as a threshold? maybe to allow for some noise?